server.go

```diff
@@ -390,7 +390,6 @@ func (s *Server) StartSelfMonitoring(database, retention string, interval time.D
 
 			// Shard-level stats.
 			tags["shardID"] = strconv.FormatUint(s.id, 10)
-			s.mu.RLock()
```

Comment:
Why are you removing these locks? They are need to avoid a race condition that was flagged on Circle.